bindings/java/wasmedge-java/build.gradle

```diff
@@ -0,0 +1,41 @@
+apply plugin: 'java-library'
+apply plugin: 'checkstyle'
+
+repositories {
+    mavenCentral()
+}
+
+configurations {
+    checkstyleConfig
+}
+
+dependencies {
+    checkstyleConfig("com.puppycrawl.tools:checkstyle:8.29") { transitive = false }
+    testImplementation('junit:junit:4.13.2')
+}
+
+test {
+    systemProperty "java.library.path", "/usr/local/lib"
+    if (System.getProperty('DEBUG', 'false') == 'true') {
+        jvmArgs '-Xdebug',
+                '-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005'
+    }
+    testLogging {
+        exceptionFormat = 'full'
+    }
+
+}
+
+apply plugin: 'checkstyle'
```

Comment:
repeated line 2? Is this expected?

Reply:
I have already ensured the latest version can work, I will approve once this is resolved.